README.md

```diff
@@ -1,2 +1,60 @@
 # ra-auth-cognito
+
 An auth provider for [react-admin](https://github.com/marmelab/react-admin) which handles authentication with AWS [Cognito](https://docs.aws.amazon.com/cognito/latest/developerguide/what-is-amazon-cognito.html).
+
+This repository contains:
+
+-   The actual `ra-auth-cognito` package
+-   A simple demo app you can run locally to try out `ra-auth-cognito` with your own UserPool.
+
+## The `ra-auth-cognito` package
+
+-   Please have a look at the [DOCUMENTATION](./packages/ra-auth-cognito/Readme.md)
+-   And also the [source code](https://github.com/marmelab/ra-auth-cognito/tree/main/src/packages/ra-auth-cognito)
+
+## Simple Demo
+
+### Prerequesites
+
+-   You need to have a [Cognito UserPool](https://docs.aws.amazon.com/cognito/latest/developerguide/what-is-amazon-cognito.html) set up.
+
+### Initial setup
+
+1. Clone this project
+
+We need to add some minimal configuration to our UserPool instance to use it. This need to be done from the AWS Console.
+
+1. Go to the [AWS Console](https://eu-west-3.console.aws.amazon.com/cognito/v2/idp/user-pools/)
+1. Select your User Pool (or create a new one)
```

Comment:
I'm stuck because by default the User Pool set the `email` attribute as required. You should add in your guide to explicitly set no attribute as required IMO.

Reply:
Also, you should suggest to enable the `name` and `picture` attributes, because it would better demonstrate the `authProvider.getIdentity()` feature.